src/ngraph/pass/manager.hpp

```diff
@@ -65,4 +73,7 @@ class ngraph::pass::Manager
     ManagerState m_state;
     bool m_visualize = false;
     bool m_serialize = false;
+    std::unordered_map<std::type_index,
```

Comment:
I wonder if I should just get rid of ManagerState if nobody is going to use it. You could put the ops_to_detail map in there.